### PR TITLE
Update sites.yml

### PR DESCRIPTION
**Pull request by r3k2**
Add 3 new sites with less that 512kb
- https://gtmetrix.com/reports/hispagatos.org/6ljRXwyp/
- https://gtmetrix.com/reports/2600.madrid/NisSVMtR/
- https://gtmetrix.com/reports/rek2.hispagatos.org/64UmBsld/

This PR is:

- [X] Adding a new domain
- [ ] Updating existing domain **size**
- [ ] Changing domain name
- [ ] Removing existing domain from list
- [ ] Website code changes (512kb.club site)
- [ ] Other not listed

- [X] I used the uncompressed size of the site
- [X] I have included a link to the GTMetrix report
- [X] The domain is in the correct alphabetical order
- [X] This site is not an ultra lightweight site
- [X] The following information is filled identical to the data file

***I confirm that I have read the [FAQ section](https://512kb.club/faq), particularly the two red items around minimal pages and inappropriate content, and I attest that this site is neither of these things.***

- [X] Check to confirm

```
- domain: hispagatos.org
  url: https://hispagatos.org
  size: 256
  last_checked: 2023-11-06
```
```
- domain: 2600.madrid
  url: https://2600.madrid
  size: 1.64
  last_checked: 2023-11-06
```
```
- domain: rek2.hispagatos.org
  url: https://rek2.hispagatos.org
  size: 2.21
  last_checked: 2023-11-06
```

GTMetrix Report: 

- https://gtmetrix.com/reports/hispagatos.org/6ljRXwyp/
- https://gtmetrix.com/reports/2600.madrid/NisSVMtR/
- https://gtmetrix.com/reports/rek2.hispagatos.org/64UmBsld/

### CODE REVIEW

**Comment by r3k2** [comment]
says is failing on "- domain: kaki87.net" I did not add this domain

File: _data/sites.yml [hunk 1]
```diff
@@ -2546,7 +2561,7 @@
   url: https://revuo-xmr.com/
   size: 444
   last_checked: 2022-04-10
-
+4
```
Comment (garritfra) [inline _data/sites.yml:2564]:
```suggestion

```

File: _data/sites.yml [hunk 2]
```diff
@@ -1676,7 +1686,7 @@
   url: https://jundimubarok.com/
   size: 58.8
   last_checked: 2022-05-05
-
+
```
Comment (garritfra) [inline _data/sites.yml:1689]:
```suggestion

```

File: _data/sites.yml [hunk 3]
```diff
@@ -1427,7 +1437,7 @@
   url: https://hund.tty1.se/
   size: 17.8
   last_checked: 2022-03-11
-
+
```
Comment (garritfra) [inline _data/sites.yml:1440]:
```suggestion

```

File: _data/sites.yml [hunk 4]
```diff
@@ -1382,6 +1387,11 @@
   url: https://herman.bearblog.dev/
   size: 6.5
   last_checked: 2022-09-12
+  
+- domain: hispagatos.org
+  url: https://hispagatos.org
+  size: 256
```
Comment (garritfra) [inline _data/sites.yml:1393]: Uncompressed size should be used.

```suggestion
  size: 405
```

File: _data/sites.yml [hunk 5]
```diff
@@ -33,6 +33,11 @@
   size: 59.6
   last_checked: 2022-05-06
 
+- domain: 2600.madrid
+  url: https://2600.madrid
+  size: 1.64
```
Comment (garritfra) [inline _data/sites.yml:38]:
```suggestion
  size: 4.97
```

File: _data/sites.yml [hunk 6]
```diff
@@ -2537,6 +2547,11 @@
   size: 32.1
   last_checked: 2022-09-18
 
+- domain: rek2.hispagatos.org
+  url: https://rek2.hispagatos.org
+  size: 2.21
```
Comment (garritfra) [inline _data/sites.yml:2552]:
```suggestion
  size: 7.93
```

**Review by garritfra — CHANGES_REQUESTED** [review]

**Review by garritfra — APPROVED** [review]

**Comment by garritfra** [comment]
Thanks! 🙂

**Pull request merged**